eng/Versions.props

```diff
@@ -1,9 +1,9 @@
 <Project>
   <PropertyGroup Label="Version settings">
     <!-- MSTest version -->
-    <VersionPrefix>3.9.0</VersionPrefix>
+    <VersionPrefix>3.10.0</VersionPrefix>
```

Comment:
will revert after, I think this PR will fail integration tests, so I don't want to revert immediately.